src/main/java/net/dries007/tfc/common/container/BarrelContainer.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Does this need an implementation/override of `moveStack`, or does the default behavior work well enough for shift clicking?